instrumentation/grape/test/test_helper.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'bundler/setup'
+Bundler.require(:default, :development, :test)
+
+require 'minitest/autorun'
+require 'webmock/minitest'
+require 'rspec/mocks/minitest_integration'
+
+# global opentelemetry-sdk setup:
+EXPORTER = OpenTelemetry::SDK::Trace::Export::InMemorySpanExporter.new
+span_processor = OpenTelemetry::SDK::Trace::Export::SimpleSpanProcessor.new(EXPORTER)
+
+OpenTelemetry::SDK.configure do |c|
+  c.error_handler = ->(exception:, message:) { raise(exception || message) }
+  c.logger = Logger.new($stderr, level: ENV.fetch('OTEL_LOG_LEVEL', 'fatal').to_sym)
+  c.add_span_processor span_processor
+end
+
+# Helper functions
+def uninstall_and_cleanup
+  instrumentation.instance_variable_set('@installed', false)
+  unsubscribe
+  EXPORTER.reset
+end
+
+def unsubscribe
```

Comment:
For a complete cleanup, it was necessary to fully unsubscribe from all built in `ActiveSupport::Notifications`. If I didn't do this, install would run multiple times because of the way the tests need to be set up and are randomised, and this created multiple subscriptions. A span was generated for each of these subscriptions.
I felt that this was the correct approach since in real scenarios, the installation will only happen once.